src/cowboy_http2.erl

```diff
@@ -586,8 +586,8 @@ commands(State=#state{socket=Socket, transport=Transport, http2_machine=HTTP2Mac
 		cow_http2:window_update(Size),
 		cow_http2:window_update(StreamID, Size)
 	]),
-	HTTP2Machine1 = cow_http2_machine:update_window(Size, HTTP2Machine0),
-	HTTP2Machine = cow_http2_machine:update_window(StreamID, Size, HTTP2Machine1),
+	HTTP2Machine1 = cal_and_update_window(Size, HTTP2Machine0),
+	HTTP2Machine = cal_and_update_window(StreamID, Size, HTTP2Machine1),
```

Comment:
That can't be right, you send the window update frames with `Size` and then update the state with the calculated size.